shared/actions/profile.js

```diff
@@ -185,13 +320,83 @@ function submitRevokeProof (proofId: string): AsyncAction {
   }
 }
 
+function checkProof (): AsyncAction {
+  return (dispatch, getState) => {
+    dispatch(updateError(null))
+    const sigID = getState().profile.sigID
+
+    // The initial checkProof() call happens while we're in the middle of the incomingCallMap flow. That can error out
+    // and we can call this again. This if case is when we're in the incomingCallMap
+    if (outputInstructionsResponse) {
```

Comment:
Hmm, I don't quite understand this.

Does this if statement only happen when this is called from the incomingCallMap flow?